editor/src/clj/editor/workspace.clj

```diff
@@ -550,6 +550,29 @@ ordinary paths."
                                                  (nil? file) (pair :missing uri)))))
                                      (iutil/group-into {} [] key val))
         notifications (notifications workspace)]
+    ;; Per-library min-version notifications
+    (let [min-version-states (into [] (filter #(= :defold-min-version (:reason %))) lib-states)
+          failing-uris (into #{} (map :uri) min-version-states)
+          all-uris (into #{} (map :uri) lib-states)
+          notification-id (fn [uri] [::dependencies-min-version uri])]
+      (doseq [{:keys [uri required current]} min-version-states]
+        (notifications/show!
+          notifications
+          {:id (notification-id uri)
+           :type :error
+           :message (localization/message
+                      "notification.fetch-libraries.min-version.error"
+                      {"uri" (str uri)
+                       "required" (str required)
+                       "current" (str current)})
+           :actions [{:message (localization/message "notification.fetch-libraries.dependencies-error.action.open-game-project")
+                      :on-action #(ui/execute-command
+                                    (ui/contexts (ui/main-scene))
+                                    :file.open
+                                    "/game.project")}]}))
+      ;; Close for those no longer failing
+      (doseq [uri (set/difference all-uris failing-uris)]
+        (notifications/close! notifications (notification-id uri))))
```

Comment:
Perhaps we should only use a single notification for library version errors? It seems to me that `(set/difference all-uris failing-uris)` is not enough in this scenario:
1. A user adds a dependency with an invalid min version -> notification is shown
2. The user removes this dependency -> notification is **not** hidden because it's not in `all-uris` at all since it's removed.

Reply:
fixed